modules/sdk-core/src/account-lib/mpc/tss/ecdsa/ecdsa.ts

```diff
@@ -225,15 +229,18 @@ export default class Ecdsa {
       let pk = getPaillierPublicKey(n);
       const k = hexToBigInt(shareToBeSend['k']);
 
-      const beta0 = Ecdsa.curve.scalarRandom();
-      shareParticipant.beta = bigIntToBufferBE(Ecdsa.curve.scalarReduce(Ecdsa.curve.scalarNegate(beta0)), 32).toString(
-        'hex'
-      );
+      const beta0 = bigintCryptoUtils.randBetween(n / _3n - _1n);
+      shareParticipant.beta = bigIntToBufferBE(
+        Ecdsa.curve.scalarReduce(Ecdsa.curve.scalarNegate(Ecdsa.curve.scalarReduce(beta0))),
+        32
+      ).toString('hex');
       const alpha = pk.addition(pk.multiply(k, hexToBigInt(shareParticipant.gamma)), pk.encrypt(beta0));
       shareToBeSend.alpha = bigIntToBufferBE(alpha, 32).toString('hex');
 
-      const nu0 = Ecdsa.curve.scalarRandom();
-      shareParticipant.nu = bigIntToBufferBE(Ecdsa.curve.scalarNegate(nu0), 32).toString('hex');
+      const nu0 = bigintCryptoUtils.randBetween(n / _3n - _1n);
```

Comment:
Existing init tests is already testing the validity of the signing process, no need for additional test. Max Int based on value [here](https://github.com/data61/python-paillier/blob/7d9911eb03c3c2d64399bc15405feb5e628379d1/phe/paillier.py#L90)